Examples/SpeechRecognition/SpeechRecognitionTests/SpeechRecognitionTests.swift

```diff
@@ -8,18 +8,15 @@ class SpeechRecognitionTests: XCTestCase {
   let recognitionTaskSubject = PassthroughSubject<SpeechRecognitionResult, SpeechClient.Error>()
 
   func testDenyAuthorization() {
-    var speechClient = SpeechClient.unimplemented
-    speechClient.requestAuthorization = { Effect(value: .denied) }
-
     let store = TestStore(
       initialState: AppState(),
       reducer: appReducer,
-      environment: AppEnvironment(
-        mainQueue: .immediate,
-        speechClient: speechClient
-      )
+      environment: .unimplemented
```

Comment:
moving towards the pattern of always defaulting to unimplemented and then overriding environment down below.